docs/root/start/sandboxes/route-mirror.rst

```diff
@@ -0,0 +1,146 @@
+.. _install_sandboxes_route_mirror:
+
+Route mirroring policies
+========================
+
+.. sidebar:: Requirements
+
+   .. include:: _include/docker-env-setup-link.rst
+
+This simple example demonstrates Envoy's request mirroring capability using
+`request mirror policies <https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/route/v3/route_components.proto#envoy-v3-api-msg-config-route-v3-routeaction-requestmirrorpolicy>`__.
+
+Incoming requests are received by ``envoy-front-proxy`` service. The envoy instance running
+in this container is configured with two routes (:download:`front-envoy.yaml <_include/route-mirror/front-envoy.yaml>`):
+
+.. literalinclude:: _include/route-mirror/front-envoy.yaml
+   :language: yaml
+   :lines: 16-34
+   :linenos:
+   :emphasize-lines: 5-17
+   :caption: Envoy configuration with route mirror policy
+
+A request for the path ``/service/1`` is forwarded to the ``service1`` cluster.
+In addition, the request is also forwarded to the ``service1-mirror`` cluster.
+
+A request for the path ``/service/2`` is forwarded to the ``service2`` cluster.
+If a header ``x-mirror-cluster`` is specified in the request, envoy extracts the
+header value and forwards the request to a cluster with the same name, if found.
+For example, if we send a header with a request as ``x-mirror-cluster: service2-mirror``,
+the request will be forwarded to the ``service2-mirror`` cluster.
+
+Step 1: Build the sandbox
+*************************
+
+Change to the ``examples/route-mirror`` directory.
+
+.. code-block:: console
+
+    $ pwd
+    envoy/examples/route-mirror
+    $ docker-compose build
+    $ docker-compose up -d
+
+.. code-block:: console
+
+    $ pwd
+    envoy/examples/route-mirror
+    $ docker-compose ps
+    NAME                               COMMAND                  SERVICE             STATUS              PORTS
+    route-mirror-envoy-front-proxy-1   "/docker-entrypoint.…"   envoy-front-proxy   running             0.0.0.0:10000->10000/tcp, :::10000->10000/tcp
+    route-mirror-service1-1            "python3 /code/servi…"   service1            running (healthy)
+    route-mirror-service1-mirror-1     "python3 /code/servi…"   service1-mirror     running (healthy)
+    route-mirror-service2-1            "python3 /code/servi…"   service2            running (healthy)
+    route-mirror-service2-mirror-1     "python3 /code/servi…"   service2-mirror     running (healthy)
+
+Step 2: Make a request to the statically mirrored route
+*******************************************************
+
+Send a request to ``service1``:
+
+.. code-block:: console
+
+  $ pwd
+  envoy/examples/route-mirror
```

Comment:
you can remove these lines

Reply:
I have removed them from the `curl` commands, but kept them for the `docker-compose` commands, as they have to be run from the specific directory.

Reply:
we only need this once at the beginning of the sandbox - we can assume that the user is in the correct dir from there on - its kinda pseudo code anyway